google-cloud-compute-v1/samples/acceptance/pagination_test.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../pagination"
+require_relative "helper"
+
+
+class ComputeDefaultValuesTest < Minitest::Test
+  PROJECT = "windows-sql-cloud"
```

Comment:
I assume successful test runs would require credentials for this project? Our CI is run against the project `helical-zone-771` (as in, the CI job has credentials for that project) so we probably need to use that project and set up test fixtures in that project instead. So the fact that CI appears to be running tests and succeeding, is puzzling to me. Can you explain what's going on?

Reply:
windows-sql-cloud is a special project which stores public SQL Server images. It should be available to any GCP project. PTAL here: https://cloud.google.com/compute/docs/images/os-details#sql_server

Reply:
Yes, it should remain accessible and tests continue to pass. I used it following the python samples.

Reply:
ok.